include/deal.II/matrix_free/evaluation_kernels_common.h

```diff
@@ -0,0 +1,99 @@
+// ------------------------------------------------------------------------
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+// Copyright (C) 2025 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// Part of the source code is dual licensed under Apache-2.0 WITH
+// LLVM-exception OR LGPL-2.1-or-later. Detailed license information
+// governing the source code and code contributions can be found in
+// LICENSE.md and CONTRIBUTING.md at the top level directory of deal.II.
+//
+// ------------------------------------------------------------------------
+
+
+#ifndef dealii_matrix_free_evaluation_kernels_common_h
+#define dealii_matrix_free_evaluation_kernels_common_h
+
+#include <deal.II/base/config.h>
+
+#include <deal.II/base/utilities.h>
+
+#include <deal.II/matrix_free/fe_evaluation_data.h>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+
+namespace internal
+{
+  template <int dim, int fe_degree, typename Number, bool is_face>
+  inline void
+  truncated_tensor_fill_up(const unsigned int n_components,
```

Comment:
I would have intuitively used another name, e.g.
```suggestion
  embed_truncated_into_full_tensor_product(const unsigned int n_components,
```
This might also not be optimal, but I think a name reflecting the embedding of a truncated tensor product representation into a regular tensor product representation (with zeros for the coefficients not present in the tensor product) would be good. We might also add a short comment.